numpy/lib/_function_base_impl.py

```diff
@@ -802,6 +802,12 @@ def select(condlist, choicelist, default=0):
     >>> np.select(condlist, choicelist, 55)
     array([ 0,  1,  2,  3,  4, 25])
 
+    >>> # Test case below showing when the default value is used
```

Comment:
The sentence should be on its own line (may need blank line after) and not written as a comment.  That will look cleaner in the result.

It would also be good to rephrase to explain a bit more about why/where.  This way the user has to dig into understanding that one point in `condlist` isn't covered and that this leads to to the 70.
(At least the second block could use an explanation as well TBH.  It wants to tell us what happens if both conditions are true, but it doesn't point that out!)